schemas/Clinical/index.js

```diff
@@ -141,6 +143,25 @@ const typeDefs = gql`
     donorId: String!
   }
 
+  type ClinicalSubmissionError implements ClinicalEntityError @cost(complexity: 5) {
+    type: String!
+    message: String!
+    row: Int!
+    field: String!
+    value: String!
+    donorId: String!
+  }
+
+  type ClinicalSubmissionSchemaError implements ClinicalEntityError @cost(complexity: 10) {
```

Comment:
added query cost to this, making it 10 since there's a bit more wrangling to get these compared to `ClinicalSubmissionError`